robottelo/helpers.py

```diff
@@ -272,6 +297,43 @@ def remove_katello_ca(hostname=None):
                 raise AssertionError('Failed to reset the rhsm.conf')
 
 
+def remove_capsule_katello_ca(hostname=None, capsule=None):
```

Comment:
same as above and delete this function once the movement is done.

Reply:
@jyejare it's moved now